server.py

```diff
@@ -50,6 +50,17 @@ class Meta:
         database = db
 
 
+class UploadedQuestion(Model):
+    question_text = TextField()
```

Comment:
is this the test case input?

Reply:
Yes this is the Test Case input

Reply:
I have changed it to a more meaningful field name **test_case_input**